src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/CloseActionIT.kt

```diff
@@ -0,0 +1,52 @@
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.action
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.IndexStateManagementRestTestCase
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.Policy
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.State
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.model.action.CloseActionConfig
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.randomDefaultNotification
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.Locale
+
+class CloseActionIT : IndexStateManagementRestTestCase() {
+
+    fun `test basic`() {
+        val indexName = "${javaClass.simpleName.toLowerCase(Locale.getDefault())}_index"
```

Comment:
Let's change this to ```toLowerCase(Locale.ROOT)```, I believe ```Locale.getDefault()``` is called implicitly when no arguments are supplied and could lead to a format issue depending on region.